protocol/v2/ssv/runner/proposer.go

```diff
@@ -237,7 +240,8 @@ func (r *ProposerRunner) ProcessPostConsensus(logger *zap.Logger, signedMsg *spe
 		logger.Info("✅ successfully submitted block proposal",
 			fields.Slot(signedMsg.Message.Slot),
 			fields.Height(r.BaseRunner.QBFTController.Height),
-			fields.Round(r.GetState().RunningInstance.State.Round))
+			fields.Round(r.GetState().RunningInstance.State.Round),
+			fields.Blinded(decidedBlindedBlock))
```

Comment:
doesn't `decidedBlindedBlock` says whether it's decided or not? `fields.Blinded` is about blinded or not.

Reply:
It says if the decided value has a blinded or regular block (also it should be called only after the instance has decided). The name is copied from the spec, so I don't want to change it only here

Reply:
Although I changed the variable name to be more clear